node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -62,7 +62,7 @@ func (r OsDeviceConnectivityHelperScsiGeneric) RescanDevices(lunId int, arrayIde
 
 		defer f.Close()
 
-		scanCmd := fmt.Sprintf("0 0 %d", lunId)
+		scanCmd := fmt.Sprintf("- - %d", lunId)
```

Comment:
r u sure it also good for A9000? to check it with iscsi.

Reply:
I think the answer is yes.
From https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/scanning-storage-interconnects
you can find that:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/scanning-storage-interconnects

Since we have no A9k in our env, so we can not do the verification

Reply:
ok so please run staging a9000 on CI before merging, to make sure u don't break a9000.

Reply:
any update?